src/pages/landing/ebooks/crypto.tsx

```diff
@@ -57,37 +58,29 @@ const StocksEbook = ({ language }: StocksEbookProps) => {
     return (
         <Layout type="landing-page" is_ppc_redirect={true}>
             <SEO
-                title={localize('Cryptocurrencies Ebook')}
-                description={localize('Trade Cryptocurrencies on our Deriv platform.')}
+                title={localize('_t_Cryptocurrencies Ebook_t_')}
+                description={localize('_t_Trade Cryptocurrencies on our Deriv platform._t_')}
                 no_index
             />
             <Hero
-                authorDesc={
-                    localize(
-                        'This e-book has been brought to you by a veteran online trader and New York Times bestselling author,',
-                    ) + ' '
-                }
-                authorName={localize('Vince Stanzione.')}
+                authorDesc={`_t_This e-book has been brought to you by a veteran online trader and New York Times bestselling author, _t_`}
```

Comment:
there is an extra space in this, please remove it so we get the same key as before. thanks :pray: